File: 1752. Check if Array Is Sorted and Rotated/main.go
package main

func check(nums []int) bool {
	cnt := 0
	for i := 1; i < len(nums); i++ {
		if nums[i] < nums[i-1] {
			cnt++
		}
		if cnt == 2 {
			return false
		}
	}
	if cnt == 1 && nums[0] < nums[len(nums)-1] {
		return false
	}
	return true
}

File: 1752. Check if Array Is Sorted and Rotated/main.py
from typing import List


class Solution:
    def check(self, nums: List[int]) -> bool:
        cnt = 0

        for i in range(1, len(nums)):
            if nums[i] < nums[i - 1]:
                cnt += 1
            if cnt == 2:
                return False   

        if cnt == 1 and nums[0] < nums[-1]:
            return False        

        return True
        